audit-ci.jsonc

```diff
@@ -3,5 +3,7 @@
 	"$schema": "https://github.com/IBM/audit-ci/raw/main/docs/schema.json",
 	"low": true,
 	"allowlist": [
+		// Taffdb is only used during jsdoc build and contains only data for jsdoc generation. After jsdoc build finished, db is destroyed
```

Comment:
still the wrong package name. Maybe also a link to the jsdoc issue would help here.

Reply:
Addressed here: https://github.com/SAP/ui5-tooling/pull/768